deploy/Gruntfile.js

```diff
@@ -65,6 +77,22 @@ module.exports = function(grunt) {
         grunt.file.write(configJSPath, configJS);
     });
 
+    grunt.registerTask('update-electron-config', function() {
```

Comment:
Where are we using this task?

Reply:
My question is that I couldn't find its usage, is it on some yaml in the pipeline? or is that the next step?

Reply:
No where yet, we will use it for the electron release pipeline